src/target.d

```diff
@@ -199,8 +199,13 @@ struct Target
         }
     }
 
-    /*
-     * Return true if the given type is supported for this target
+    /**
+     * Checks whether the target supports a vector type with total size `sz`
+     * (in bytes) and element type `type`.
+     *
+     * Returns: 0 if the type is supported, or else: 1 if vector types are not
+     *     supported on the target at all, 2 if the given size isn't, or 3 if
+     *     the element type isn't.
```

Comment:
So, the question is: should LDC return 0 if the target doesn't support e.g. 256-bit SIMD instructions, but the backend can emulate them with e.g. 2 128-bit instructions. If that's the case, I think it's a good idea for this information to be included in the comment.
Otherwise, disregard my question.